.stylelintrc.json

```diff
@@ -0,0 +1,102 @@
+  "plugins": [
+    "stylelint-order"
+  ],
+  "ignoreFiles": [
+    "components/demo/code-dark-plus-styles.js",
+    "components/demo/code-tomorrow-styles.js",
+    "components/list/**/*.js"
+  ],
+  "rules": {
+    "at-rule-name-case": "lower",
+    "at-rule-name-space-after": "always-single-line",
+    "at-rule-semicolon-newline-after": "always",
+    "block-closing-brace-newline-after": "always",
+    "block-closing-brace-newline-before": "always-multi-line",
+    "block-closing-brace-space-before": "always-single-line",
+    "block-opening-brace-newline-after": "always-multi-line",
+    "block-opening-brace-space-after": "always-single-line",
+    "block-opening-brace-space-before": "always",
+    "color-hex-case": "lower",
+    "color-hex-length": "long",
+    "comment-whitespace-inside": "always",
+    "custom-property-pattern": [ "^d2l-", {
+      "message": "Expected custom property to begin with \"d2l-\""
+    } ],
+    "custom-property-empty-line-before": [ "always", {
+      "except": [
+        "after-custom-property",
+        "first-nested"
+      ],
+      "ignore": [
+        "after-comment",
+        "inside-single-line-block"
+      ]
+    } ],
+    "declaration-bang-space-after": "never",
+    "declaration-bang-space-before": "always",
+    "declaration-block-semicolon-newline-after": "always-multi-line",
+    "declaration-block-semicolon-space-after": "always-single-line",
+    "declaration-block-semicolon-space-before": "never",
+    "declaration-block-single-line-max-declarations": 3,
+    "declaration-block-trailing-semicolon": "always",
+    "declaration-colon-newline-after": "always-multi-line",
+    "declaration-colon-space-after": "always-single-line",
+    "declaration-colon-space-before": "never",
+    "function-comma-newline-after": "always-multi-line",
+    "function-comma-space-after": "always-single-line",
+    "function-comma-space-before": "never",
+    "function-max-empty-lines": 0,
+    "function-name-case": "lower",
+    "function-parentheses-newline-inside": "always-multi-line",
+    "function-parentheses-space-inside": "never-single-line",
+    "function-whitespace-after": "always",
+    "indentation": "tab",
+    "length-zero-no-unit": true,
+    "max-empty-lines": 2,
+    "media-feature-colon-space-after": "always",
+    "media-feature-colon-space-before": "never",
+    "media-feature-name-case": "lower",
+    "media-feature-parentheses-space-inside": "never",
+    "media-feature-range-operator-space-after": "always",
+    "media-feature-range-operator-space-before": "always",
+    "media-query-list-comma-newline-after": "always-multi-line",
+    "media-query-list-comma-space-after": "always-single-line",
+    "media-query-list-comma-space-before": "never",
+    "no-descending-specificity": null,
+    "no-eol-whitespace": [ true, {
+      "ignore": [ "empty-lines" ]
+    } ],
+    "number-leading-zero": "always",
+    "number-no-trailing-zeros": true,
+    "order/order": [
```

Comment:
These two rules configure the order.  Optionally, we can specify an order other than alphabetic.